cpp/ql/test/library-tests/syntax-zoo/aliased_ssa_sanity.expected

```diff
@@ -20,6 +20,7 @@ instructionWithoutSuccessor
 | ms_try_mix.cpp:11:12:11:15 | Chi: call to C |
 | ms_try_mix.cpp:28:12:28:15 | Chi: call to C |
 | ms_try_mix.cpp:48:10:48:13 | Chi: call to C |
+| pointer_to_member.cpp:35:11:35:21 | FieldAddress: {...} |
```

Comment:
It's true that the IR can't be made to handle pointers to members correctly in all cases, but an `instructionWithoutSuccessor` failure usually means that lots of unrelated code gets deleted because it's determined to be unreachable. That's probably worth fixing.

Reply:
Sure. Once this workaround is merged, I'll try get get PTMs working as best I can.

Reply:
Sure. Once this workaround is merged, I'll try get get PTMs working as best I can.